### DISCUSSION TITLE
shihongxi有后台权限？

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#125] [Q&A] [answered]
为什么他在讨论被锁定后又发送了一条消息？ [tid=222](http://www.xmoj.tech/discuss3/thread.php?tid=222)

![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/81672884/1b581b7f-a9b6-45e2-bc6b-43599296b836)

**Comment by langningchen** [comment]
如果真的有，那他应该要在前端加上 `管理员` Badge

**Comment by PythonSmall-Q** [comment]
不可能，他不是管理员

Sent from Mail for Windows

From: Chen LangNing
Sent: 2023年10月8日 22:10:36
To: XMOJ-Script-dev/XMOJ-Script
Cc: Subscribed
Subject: [XMOJ-Script-dev/XMOJ-Script] shihongxi有后台权限？ (Discussion #125)

为什么他在讨论被锁定后又发送了一条消息？ tid=222

**Reply by langningchen** [reply]
那他是如何在讨论锁定后又发消息？

**Reply by langningchen** [reply]
@PythonSmall-Q

**Comment by PythonSmall-Q** [comment]
不知道，我也很懵逼

Sent from Mail for Windows

**Comment by langningchen** [comment]
你们检查一下后台有没有这种安全漏洞 @PythonSmall-Q @boomzero

**Reply by PythonSmall-Q** [reply, marked as answer]
我等有空了看看你那快两万行的代码，我也会和他核实一下

**Reply by PythonSmall-Q** [reply]
《毒瘤》 @langningchen

**Comment by langningchen** [comment]
好吧
